alpha/apps/kaltura/lib/storage/kStorageExporter.php

```diff
@@ -142,7 +171,7 @@ public static function reExportEntry(entry $entry)
 			$scope = $profile->getScope();
 			$scope->setEntryId($entry->getId());
 			if($profile->triggerFitsReadyAsset($entry->getId()) && $profile->fulfillsRules($scope))
-				self::exportEntry($entry, $profile);
+				self::tryExportEntry($entry, $profile);
```

Comment:
StorageProfilePeer::retrieveExternalByPartnerId also returns only enabled profiles, no need to try and catch.